pyproject.toml

```diff
@@ -50,15 +50,21 @@ proxy = [
     "pyyaml",
     "rq",
     "orjson",
+    "tenacity",
+    "cohere"
 ]
 
 extra_proxy = [
     "prisma",
-    "azure-identity",
-    "azure-keyvault-secrets",
-    "google-cloud-kms",
     "streamlit",
-    "resend"
+    "redis",
+    "google-generativeai",
+    "google-cloud-aiplatform",
+    "langchain",
+    "langfuse",
+    "anthropic",
+    "traceloop-sdk",
+    "boto3"
 ]
```

Comment:
@ishaan-jaff I a bit unsure here regarding the essential and optional dependencies.

Reply:
@bufferoverflow any reason you need this here (client sdk) ?

The dockerfile covers this with the requirements.txt - https://github.com/BerriAI/litellm/blob/main/requirements.txt

Reply:
The dependencies where here https://github.com/BerriAI/litellm/blob/main/.circleci/requirements.txt , but I will rework this MR here to reflect the requirements.txt you linked.

Reply:
i guess the question is - why have it be in the pyproject.toml?